### PR TITLE
made files lowercase

### PR DESCRIPTION
**Pull request by JustDoom**
# Description

made the files lowercase and fixed some links to other files

## Checklist

- [x] My contributions follows the style guidelines of this project
- [x] I have performed a self-review of my own contributions (checked for spelling errors, proper linking, etc)
- [x] I have run linting on my branch before submitting the PR

### CODE REVIEW

File: server/guides/community/community.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Community
+
+A community is key for a Minecraft server. It keeps players connected to your server. It also makes your server a fun place. A community is not for sale, but it is worth a lot. You build a community and have to maintain a community as a true leader. We&#39;ll discuss three things in this guide: building a community, maintaining a community and some general tips.
+
+Before discussing how to build and maintain a community, it&#39;s important to understand why people become part of a community and what they are looking for there. With this basis in mind, we can then look at how you can set up a community yourself.
```
Comment (Silverarmor) [inline server/guides/community/community.md:5]: `&#39` - is this following the style convetions of the project?

File: server/guides/community/community.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Maintaining your community
+
+A community must remain &quot;alive&quot;. That means there is activity, members talk to each other, come up with ideas and do things. When a community comes to a standstill, it is very difficult to get the community going again. So always make sure that your community stays going.
+
+One of the best ways to keep members engaged is events. This can be incredibly lengthy and doesn&#39;t even have to be related to Minecraft. As long as members interact with each other
+
+To keep a community a fun place, there should be as few &quot;bad apples&quot; in your community as possible. These are members who don&#39;t want the best for the community. Think of toxic members, trolls, bullies, and so on. This is best done by showing individual members that they are responsible for their community, but at the same time, it is very difficult. And on top of that, the more members a community has, the less likely the individual will feel responsible.
```
Comment (Silverarmor) [inline server/guides/community/community.md:25]: > `&quot;bad apples&quot;`

File: server/guides/moderation/moderation.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Moderating a Minecraft server
+
+Just as server settings are subject to debate, so is moderating a server. Everyone does it in their own way. But how efficient and good your server settings are is measurable. How well you moderate is much less measurable. Therefore, please see this part of the guide as well-intentioned advice and not as hard rules.
+
+## Rules and sanctions
+
+Before you can decide what rules to implement, it&#39;s important to determine what kind of server you want to run. After all, a family-friendly server for children has very different rules than an anarchy server. Often you will be looking for a balance between player freedom and player safety. For most servers, a good basis to adhere to is: a player&#39;s freedom stops when other players are bothered by actions.
```
Comment (Silverarmor) [inline server/guides/moderation/moderation.md:7]: > `&#39`

Still present. Have you lintd these changes?

**Review by Silverarmor — CHANGES_REQUESTED** [review]

File: server/guides/moderation/moderation.md (hunk 3 above)
Comment (JustDoom) [reply server/guides/moderation/moderation.md]: i did. im probably running it wrong though

**Review by Silverarmor — APPROVED** [review]

**Pull request merged**